ejb/WFLY-13090-Enable-compression-on-remoting-globally-for-EJB-calls.adoc

```diff
@@ -0,0 +1,80 @@
+= Enable compression on remoting globally for EJB calls
+:email:             tomasz.adamski@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+=== Issue
+* https://issues.redhat.com/browse/WFLY-13090
+
+=== Dev Contacts
+
+* mailto:tadamski@redhat.com
+
+=== QE Contacts
+
+=== Testing By
+[ ] Engineering
+
+[x] QE
```

Comment:
@tadamski EAP7-1202 says testing is by Engineering - so one of these must be wrong :)

Reply:
@kabir yeah - this was an error - fixed, thanks